war/src/main/webapp/css/style.css

```diff
@@ -393,7 +414,7 @@ pre.console {
 }
 
 #jenkins .task-link:hover {
-    text-decoration: underline;
+    text-decoration: none;
```

Comment:
I'm wondering if this is want we want as it will affect the visualization of the link